chapters/connectors.tex

```diff
@@ -903,7 +905,19 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
-Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
+Only the overdetermined connector components are considered.
+This generation replaces the equality-equation for variables that are neither flow nor stream in \cref{generation-of-connection-equations}.
+
+In order to handle any redundant connect-equation the following preparation is needed.
+\begin{enumerate}
+\item
+  We build the connection sets in the normal way, keeping the overdetermined components as primitives.
+\item
+  Instead of generating the equality-equation for an overdetermined component a breakable branch in the virtual connection graph is constructed.
+\end{enumerate}
+If there are no redundant connect-equations the breakable branches will correspond to the connect-equations for overdetermined connectors, otherwise this will only generate branches for a subset of them.
```

Comment:
This is not true.  The correspondence to `connect`-equations is only one of many possibilities for which equality-equations that could be generated for a normal connection set.

Also, missing styling of "`connection`-equation".

```suggestion
If there are no redundant connect-equations the breakable branches can be chosen to correspond to the \lstinline!connect!-equations for overdetermined connectors, otherwise this will always exist \lstinline!connect!-equations without a corresponding breakable branch.
```

Reply:
The first part makes sense, the second less so.

Reply:
Have updated

Reply:
I don't mind skipping the second part.  Resolving obsolete conversation.